localization/english/bookmark/bookmark_508_l_english.yml

```
@@ -1,11 +1,11 @@
 ﻿l_english:
  bm_the_gnoll_war:0 "Gnoll War"
- bm_the_gnoll_war_desc:0 "PH"
+ bm_the_gnoll_war_desc:0 "After the collapse of Arathor annd the division of humanity into several kingdoms and city-states, settlers from Arathi migrated to the southern land of Azeroth and founded the city of Stormwind. Led by the noble Wrynn dynasty,the kingdom became a large and prosperous nation but the new king Barathen have now to face the rise of another ambitious ruler, Garfang, leader of the largest gnoll pack ever seen and determined to conquer and plunder the rich human lands. Will Stormwind maintain its place as the dominant power of the sub-continent or is it time for the mongrel gnolls to finally rule ? "
 
 
  #Barathen Wrynn
  barathen_wrynn_character:0 "King Barathen"
- barathen_wrynn_character_desc:0 "PH"
+ barathen_wrynn_character_desc:0 "After the death of his father, Landan Wrynn, king Barathen now has to rules during the gravest crisis the kingdom faced since its foundation, but he remains adamant. With strong will and discipline and the superiority of his tactics, the king knows he can defeat his barbaric foes and make of Stormwind the mightiest of human nations."
```

Also further suggestion:
- Changed the last sentence to limit the "and" conjunction a little bit.
- Shortening the first sentence by splitting them.

```suggestion
 barathen_wrynn_character_desc:0 "After the death of his father, Landan Wrynn, king Barathen now has to rule during the gravest crisis the kingdom faced since its foundation. But the king remains adamant. With strong will, discipline , and the superiority of his tactics, the king is confident that he can defeat his barbaric foes. Paving the way for Stormwind to become mightiest of all human nations."
```